### ISSUE TITLE
Both versions 5.7 and 6.0 soft-brick on 16.1.0

### ISSUE THREAD

**Issue by ThirstyWraith** [StarDustCFW/Haku33#28]
Using Atmosphere 1.6.2, Firmware 16.1.0:

- After wiping, version 5.7 causes HOS to not start whatsoever, the screen remains black
- After wiping, version 6.0 starts but you permanently receive the 2002-4373 error code and the console keeps asking you to reboot

This seems to be consistent.

**Comment by Kronos2308** [comment]
Yes seems like y put a check bad an think that the console in on 17.0.0 all I will fix that today

**Comment by Kronos2308** [comment]
to fix this just copy 
[startup.te](https://github.com/StarDustCFW/Haku33/blob/master/romfs/startup.te?raw=true)
to SD card and launch
[TegraExplorer.bin](https://github.com/StarDustCFW/Haku33/blob/master/romfs/TegraExplorer.bin?raw=true)
the the error will be gone

**Comment by jorgeperezlara** [comment]
For me this didn't work. I had to use TegraExplorer's wipe option, instead of the one from the script.

I got really scared hahaha

**Comment by Kronos2308** [comment]
@jorgeperezlara  nice you know that if you update to 17.0.0 the console will get a softbrik

**Comment by Kronos2308** [comment]
That's why I tell that use my script because the one on tegra explorer delete the index

**Comment by jeoridin** [comment]
Screen remains black and no signs of life after reset. Turned off auto rcm and I have 0 clue on how to use TegraExplorer.bin without my switch being in recovery mode. Any ideas?

**Comment by Kronos2308** [comment]
You have to turn off the auto rcm

**Comment by jeoridin** [comment]
Auto rcm is already off. 

How do I input the startup.te file? I understand that I have to load it onto the SD card but how? given that it is not a downloadable file as when I click on it it goes straight to a text document.  

For the TegraExplore.bin file do I just need to put that into the main folder of my SD card, do I need to remove any other .bin files?

**Comment by Kronos2308** [comment]
Is a Tex document because is a script 
Just right click save has 

You south learn how to properly run payloads in your switch, before try something learn the basics

**Comment by jeoridin** [comment]
How do I run the payload if I cant turn my switch on?

**Comment by Kronos2308** [comment]
Entering on RCM and loading a payload

**Comment by Ngov0** [comment]
I wiped my switch with haku33 and am having this problem. My switch's screen only turns on when I plug it in (to show the flat symbol). I cant boot it to recovery mode anymore

**Comment by Kronos2308** [comment]
you use the latest ?

**Comment by Kronos2308** [comment]
must run Atmosphere to rebuild your NCA database

**Comment by chikennoodle** [comment]
screen does not turn on anymore. 
ive put the text file into my sd card 
cant put switch into rcm so i cant run the .bin file 
I try to put switch into rcm but tegrarcm says NO RCM
same results without sd card in switch

**Comment by Kronos2308** [comment]
Try another wire
Power off the console holding power 12s
If is briked has easy solution
 But the payload loader has nothing to be with haku33

**Comment by jorgermiranda** [comment]
@Kronos2308 I've used version 6.1 (the latest at this moment), and after running the .nro, it restarts, shows the Nintendo logo, but then screen remains black.
I've try by copying the startu.te and starting the TegraExplorer.bin, the app seems to run properly, but the I got black screen after nintendo logo.

Any other update on this?

Update: I've attached the result of what I see once startup.te was executed.
![screen](https://github.com/StarDustCFW/Haku33/assets/8164555/36ed27b7-d516-4a79-9de6-80bed62ed2fd)

**Comment by xXMOOXx** [comment]
I had the same problem. Updating Atmosphere did it for me, you should do that first. Then to fix it, i had to start (run) the TegraExplorer.bin, then navigate to "browse emmc" -> "system" -> "saves" and delete all the files there. Then go back to the first folder and run atmosphere from there (im not sure how it was called but something like "run atmosphere.bin/payload". But remember to first update your Atmosphere to the latest! Then the console started and was factory reset.

**Comment by Kronos2308** [comment]
This is the same has #29 
We found 3 solution for the problem
1 just run atmosphere on Sysnand 
2 delete system:/saves/8000000000000120 using tegra explorer.bin
And then run atmosphere to rebuild db
Or 
3 delete system:/saves completely using tegra explorer.bin an the run atmosphere (this will make a full factory restart)

I I'll leave this open until whoever created it sees fit.

**Comment by Kronos2308** [comment]
I I'll leave this open until whoever created it sees fit.
